public synchronized Exception getError(){
    return null;
}